src/views/Explore.vue

```diff
@@ -56,6 +56,7 @@
         <NoResults :block="true" v-if="Object.keys(items).length < 1" />
       </div>
     </Container>
+    <div id="endsensor"></div>
```

Comment:
The smaller the better :)
```suggestion
    <div id="endsensor"/>
```